client/src/mapper-consents/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { apiErrorMessage, getMapperConsents, useApi } from '../api';
+import { BeatmapInline } from '../BeatmapInline';
+import { IMapperConsent } from '../interfaces';
+import { UserInline } from '../UserInline';
+
+function consentToCell(consent?: 0 | 1 | 2 | boolean) {
+  let index: 'null' | 0 | 1 | 2;
+  if (consent === true) {
+    index = 1
+  } else if (consent === false) {
+    index = 0
+  } else {
+    index = consent == null ? 'null' : consent;
+  }
+  const mapping = {
+    'null': ['no reply', 'pending'],
+    0: ['no', 'error'],
+    1: ['yes', 'success'],
+    2: ['unreachable', 'pending']
+  };
+  const [consentStr, className] = mapping[index];
+
+  return (
+    <td className={className}>
+      {consentStr}
+    </td>
+  )
+}
+
+function maybeBeatmapsetTable(consent: IMapperConsent) {
+  if (consent.beatmapset_consents.length === 0) {
+    return
+  }
+  return (
+    <tr key={consent.id + "-dropdown"}>
+      <td colSpan={3}>{MapperBeatmapsetConsents(consent)}</td>
+    </tr>
+  )
+}
+
+function MapperBeatmapsetConsents(mapperConsent: IMapperConsent) {
+  return (
+    <table style={{'width': '80%', 'marginLeft': '3em', 'marginRight': '3em', 'tableLayout': 'fixed'}}>
```

Comment:
I was hoping the inset from the left edge would denote that it belongs to the mapper above it but it's not great :/ my only remaining idea is to put a pink border around the table and remove the left margin and maybe that will make it clearer who it belongs to?

Reply:
I locked the column sizes because I wanted all of the tables to line up in terms of where each column was and with auto sized tables each table's column width changes depending on the length of the map's title, not sure if there's a better way to do tha

Reply:
couldn't figure out a good fix myself rn so will just see what people say about it